lib/serializer/src/operations.js

```diff
@@ -622,7 +622,16 @@ export const asset_options = new Serializer("asset_options", {
     whitelist_markets: set(protocol_id_type("asset")),
     blacklist_markets: set(protocol_id_type("asset")),
     description: string,
-    extensions: set(future_extensions)
+    extensions: extension([
```

Comment:
How do I build that part in asset_options then?

`extensions: {reward_percent: 100}` ?

Reply:
my fault. i'll write an example